src/3a_count_MPA.jl

```diff
@@ -122,11 +126,11 @@ end
 
 # Write data to shapefile (ArcGIS does not accept geopackage format)
 GDF.write(
-    joinpath(QGIS_DIR, "reef_suitability.shp"),
+    joinpath(MPA_QGIS_DIR, "reef_suitability.shp"),
     reef_features[:, [:geometry, :region, :reef_name, :flat_ha, :slope_ha, :Area_HA, :n_flat, :n_slope, :flat_scr, :slope_scr, :UNIQUE_ID]];
     layer_name="reef_suitability",
     geom_columns=(:geometry,),
-    crs=EPSG(4326)
+    crs=EPSG(7844)
 )
```

Comment:
FYI, this can now be outputted to geopackage. Figured out what I was doing wrong getting the data into ArcGIS (definitely user error)